sale_order_type/views/sale_order_type_view.xml

```diff
@@ -52,10 +52,7 @@
                                 name="pricelist_id"
                                 groups="product.group_product_pricelist"
                             />
-                            <field
-                                name="incoterm_id"
-                                groups="sale_stock.group_display_incoterm"
```

Comment:
Why are you removing the group on this field?

Reply:
This group Was removed, see https://github.com/odoo/odoo/commit/9ddc128d7bd3998fb3cf1f1c55d0d76fccc91196